samples/FileBasedMcpServer/Program.cs

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env -S dotnet run --
```

Comment:
Could it be simplified into `#!/usr/bin/env dotnet`, I don't know about the difference